src/dotlottie-player.ts

```diff
@@ -343,219 +581,256 @@ export class DotLottiePlayer extends LitElement {
+   */
+  public previous(playbackOptions?: PlaybackOptions): void {
+    this._requireAnimationsInTheManifest();
+    this._requireAnimationsToBeLoaded();
+
+    if (this._activeAnimationIndex === 0) {
+      this._activeAnimationIndex = this._manifest.animations.length - 1;
+    } else {
+      this._activeAnimationIndex -= 1;
+    }
+    this.play(this._activeAnimationIndex, playbackOptions);
+  }
+
+  /**
+   * Play the next animation. The order is taken from the manifest.
+   */
+  public next(playbackOptions?: PlaybackOptions): void {
+    this._requireAnimationsInTheManifest();
+    this._requireAnimationsToBeLoaded();
+
+    if (this._activeAnimationIndex === this._manifest.animations.length - 1) {
+      this._activeAnimationIndex = 0;
+    } else {
+      this._activeAnimationIndex += 1;
+    }
+    this.play(this._activeAnimationIndex, playbackOptions);
+  }
+
+  /**
+   * Reset to the initial state defined in the manifest.
+   */
+  public reset(): void {
+    this._requireAnimationsInTheManifest();
+    this._requireAnimationsToBeLoaded();
+
+    if (this._manifest && this._manifest['activeAnimationId']) {
+      this.play(this._manifest['activeAnimationId']);
+    } else if (this.activeAnimationId) {
+      this.play(this.activeAnimationId);
+    } else {
+      // If no active animation is specified, play the first one
+      this.play(0);
+    }
+  }
+
   /**
    * Start playing animation.
    */
-  public play() {
+  public play(targetAnimation?: string | number, playbackOptions?: PlaybackOptions): void {
     if (!this._lottie) {
       return;
     }
 
-    this._lottie.play();
-    this.currentState = PlayerState.Playing;
+    // If no animation is specified, play the current animation
+    if (targetAnimation === undefined) {
+      this._lottie.play();
+      this.currentState = PlayerState.Playing;
+
+      this.dispatchEvent(new CustomEvent(PlayerEvents.Play));
+
+      return;
+    }
+
+    this._requireAnimationsToBeLoaded();
+    this._requireAnimationsInTheManifest();
+
+    // If animation is specified, check if it exists and play it
+    if (targetAnimation !== undefined) {
+      if (!this._animations) throw createError(`No animations have been loaded.`);
+      this._requireAnimationsInTheManifest();
+
+      if (typeof targetAnimation === 'string') {
+        const animationIndex = this._manifest.animations.findIndex((element) => element.id === targetAnimation);
+
+        if (animationIndex !== -1) {
+          this._activeAnimationIndex = animationIndex;
+
+          if (this.src) {
+            this.load(this._animations[this._activeAnimationIndex], { playbackOptions });
+          }
+        } else {
+          logError(`No animation with the id '${targetAnimation}' was found.`);
+        }
+      } else if (typeof targetAnimation === 'number') {
+        if (this.src && this._manifest.animations && this._manifest.animations[targetAnimation]) {
```

Comment:
we need to validate if the `targetAnimation` index is within the `this._manifest.animations` boundaries